### ISSUE TITLE
Button alignment in Form = space between

### ISSUE THREAD

**Issue by YariMVSage** [Sage/carbon#5536]
### Desired behaviour

I would like to be able to set something like space between in the buttonAlignment property of the Form component. As this component is not only used to create dialogs with buttons.

### Current behaviour

Right now I can only set button alignment left or right

### Suggested Solution

In the codesandbox example you can see that the button "Previous" is not aligned. I need the previous button aligned to the left and the Next button aligned to the right.

### CodeSandbox or Storybook URL

https://codesandbox.io/s/loving-grothendieck-t5qn0t?file=/src/index.js

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by YariMVSage** [comment]
We have this problem in prod it would be nice if we can fixed it soon, if possible, thanks!

**Comment by ljemmo** [comment]
@YariMVSage please can you share a design example of the above via teams or slack so that I can review whether this needs supporting in the DS/carbon. Traditionally we tend to place the step sequence at the top of the page instead of in a footer.

**Comment by Parsium** [comment]
Hi @harpalsingh @ljemmo, are you aware of any updates on this one? @YariMVSage is this currently still blocking you in production? If so we try and find a temporary workaround for you.

**Comment by YariMVSage** [comment]
Hi @Parsium at the moment we can not place the StepSequence component at the top of the dialog, we would need to book hours to do so in our roadmap, also @ljemmo your proposal has got the footer Form buttons in the middle, something that currently it is not possible to achieve and furthermore, if we place the StepSequence component at the top, there will be some steps that we will have to refactor and break them in two as we will not have enough space for all the fields. So, no. At this moment we haven't got a workaround

**Comment by Parsium** [comment]
Hi @YariMVSage, if this is urgent and blocking your work I would get in touch with your designer as soon as possible so they can discuss this with the Design System team. Unfortunately, we cannot proceed with this unless we have confirmation from DS that this is a supported pattern.

**Comment by Parsium** [comment]
Following our conversation outside github, I'll close this issue for now as you are waiting for some redesigns. If once you have received the new designs you still aren't able to implement this using Carbon, please raise another issue with us and we'll investigate further 👍🏼

**Comment by ljemmo** [comment]
@YariMVSage please ignore the centre aligned buttons in the footer. That was a mistake on my end. 

The main point i was trying to get across in that quick mockup was to place the step flow at the top of the dialog. Carbon dialogs can have vertical scroll, so you shouldn't need to split the steps into 2 at any point FYI.
